drv/gimlet-seq-server/src/main.rs

```diff
@@ -201,9 +201,13 @@ struct ServerImpl<S: SpiServer> {
     hf: hf_api::HostFlash,
     vcore: vcore::VCore,
     deadline: u64,
+    // Buffer for encoding ereports. This is a static so that it's not on the
+    // stack when handling interrupts.
+    ereport_buf: &'static mut [u8],
```

Comment:
Nit: this is a `&mut [u8]` here and a `&mut [u8; EREPORT_BUF_LEN]` in `cosmo-seq-server`.  Any reason for the inconsistency?